app/domain/service/scheduled.go

```diff
@@ -21,6 +21,6 @@ import "github.com/limechain/hedera-eth-bridge-validator/app/model/transfer"
 // Scheduled interface is implemented by the Scheduled Service
 // Provides business logic for execution of Scheduled Transactions
 type Scheduled interface {
-	// Execute submits a scheduled transaction and executes provided functions when necessary
-	Execute(id, nativeAsset string, transfers []transfer.Hedera, onExecutionSuccess func(transactionID, scheduleID string), onExecutionFail, onSuccess, onFail func(transactionID string))
+	// ExecuteScheduledTransferTransaction submits a scheduled transaction and executes provided functions when necessary
+	ExecuteScheduledTransferTransaction(id, nativeAsset string, transfers []transfer.Hedera, onExecutionSuccess func(transactionID, scheduleID string), onExecutionFail, onSuccess, onFail func(transactionID string))
```

Comment:
Nit: maybe we will need to add `ExecuteScheduledMint` function in this interface?